Makefile

```diff
@@ -161,15 +161,19 @@ ui-build:
 ui-test: ui-build
 	./hack/test-ui.sh
 
+.PHONY: ext-build
+ext-build:
+	cd serving && $(MAKE) build
+
 .PHONY: image
-image: clean ui-build dist/$(BINARY_NAME)-linux-amd64
+image: clean ui-build dist/$(BINARY_NAME)-linux-amd64 ext-build
```

Comment:
Since we still build binaries in docker,  so we don't need `ext-build`, right?

Reply:
we copy the Cargo.lock file which will be generated after we build